portfolio/src/main/webapp/script.js

```diff
@@ -42,3 +42,24 @@ function showSlides(newIndex) {
   slides[slideIndex].style.display = "block";
   dots[slideIndex].className += " active";
 }
+
+/**
+ * Displays list of comments returned by server.
+ */
+ function showComments() {
+   fetch('/comments').then(response => response.json()).then((comments) => {
+    const commentsElement = document.getElementById('text-container');
+    commentsElement.innerHTML = '';
+    for(let i = 0; i < comments.length; i++) {
+        commentsElement.appendChild(
+        createListElement(comments[i]));
+    }
+  });
+ }
+
+ /** Creates an <li> element containing text. */
```

Comment:
Fix indentation: Here, you should not have any indentation.
Further above at the showComments method, you have an indentation of 1 whitespace which is also wrong.
2 whitespaces per indentation level.